docs/codeql/codeql-language-guides/basic-query-for-cpp-code.rst

```diff
@@ -3,7 +3,9 @@
 Basic query for C and C++ code
 ==============================
 
-Learn to write and run a simple CodeQL query using LGTM.
+Learn to write and run a simple CodeQL query using Visual Studio Code with the CodeQL extension.
+
+.. include:: ../reusables/setup-to-run-queries.rst
```

Comment:
I've added this placeholder here so that it's easy to update this as we steadily make it easier to get started with running CodeQL queries.